docker/Dockerfile

```diff
@@ -1,12 +1,16 @@
 ARG BASE_IMAGE=ubuntu:22.04
 FROM "$BASE_IMAGE" AS base
+ARG TARGETARCH
+ARG TARGETARCH="${TARGETARCH:-amd64}"
```

Comment:
I don't think dockerfile support this kind of syntax?

Reply:
Seems not, I guess we need to require `DOCKER_BUILDKIT=1` to enable those args.

Reply:
I'm OK with buildkit but it has its own issue as well, so don't dep on it for unnecessary features.